controllers/helpers.go

```diff
@@ -16,15 +16,6 @@ limitations under the License.
 
 package controllers
 
-func containsString(slice []string, s string) bool {
-	for _, item := range slice {
-		if item == s {
-			return true
-		}
-	}
-	return false
-}
-
```

Comment:
Seems like we're likely to need this at some point in the future to me, worth keeping and telling golint to ignore for now?

Reply:
it's a trivial function if we need it we can add it in the future